packages/amazonq/src/lsp/client.ts

```diff
@@ -252,6 +252,59 @@ async function initializeAuth(client: LanguageClient): Promise<AmazonQLspAuth> {
     return auth
 }
 
+// jscpd:ignore-start
```

Comment:
why we need this?

Reply:
The GitHub checks for duplicates fails, but the code is similar, but not duplicates of what it finds.